packages/fxa-admin-panel/src/components/AccountSearch/index.tsx

```diff
@@ -270,8 +283,18 @@ const AccountSearchResult = ({
   };
   query: string;
 }) => {
-  if (loading) return <p data-testid="loading-message">Loading...</p>;
-  if (error) return <p data-testid="error-message">An error occurred.</p>;
+  if (loading)
```

Comment:
Interesting. I would have expected this to fail ESLint checks, but maybe we don't have the [curly](https://eslint.org/docs/rules/curly) rule enabled as a default.